### PR TITLE
Zimperium MTD - add format

### PR DESCRIPTION
**Pull request by lvoloshyn-sekoia**
https://github.com/SekoiaLab/detection-rules/issues/3926

## Type of Change

- [x] New intake format
- [ ] Update to existing intake format
- [ ] Bug fix
- [ ] Documentation update
- [ ] Other (please describe):

## Affected Intake Formats

- Vendor/product:

### 🔒 CRITICAL - Security Requirements (MUST BE VERIFIED FIRST)

- [ ] **No real email addresses** in test files (use: user@example.com, test@example.org)
- [ ] **No real passwords or API keys** (use: "P@ssw0rd!", "xxxxxxxxxxxx", "FAKE_TOKEN_123")
- [ ] **No real IP addresses** (use TEST-NET ranges: 192.0.2.x, 198.51.100.x, 203.0.113.x)
- [ ] **No real phone numbers** (use: +1-555-0100 or similar test numbers)
- [ ] **No real names or PII** (use: "John Doe", "Jane Smith", "User123")
- [ ] **No real company/organization names** in test data (unless public vendors)
- [ ] **No real usernames, hostnames, or domain names** (use: user1, host.example.com, test.corp)
- [ ] All test data has been anonymized and verified

### Required for All PRs

- [ ] Code has been linted
    - [ ] with the linter for manifest, smart-descriptions, and test files (`uv run python linter.py check --changes` in utils/)
    - [ ] with Prettier for the parser code files (`npx prettier --write <module>/<format>/ingest/parser.yml`)
- [ ] All CI/CD checks pass
- [ ] Changes follow the contribution guidelines in `CONTRIBUTING.md`

### Required for New/Updated Intake Formats

- [ ] Clear descriptions provided for modules and formats
- [ ] Logo files included for new modules/formats
- [ ] Parser test coverage is at least 75%
- [ ] At least one smart-description is provided
- [ ] README.md updated (if applicable)
- [ ] Taxonomy mappings are correct and documented
- [ ] Test cases cover edge cases and error handling

### Testing

- [ ] Local tests pass (`uv run pytest` in utils/)
- [ ] Sample data is representative and **completely anonymized**
- [ ] **No sensitive information** (PII, credentials, real IPs, real emails) in test files
- [ ] Test data uses example.com/example.org domains and TEST-NET IP ranges
- [ ] Parser handles malformed data gracefully

### Documentation

- [ ] Code changes are documented
- [ ] Smart-descriptions are clear and informative
- [ ] Field mappings are explained

## Related Issues

Closes #

## Summary by Sourcery

Add a new intake format for Zimperium Mobile Threat Defense, including parser, metadata, and vendor module definition.

New Features:
- Introduce a JSON-based parser for Zimperium Mobile Threat Defense events mapping to ECS and custom zimperium.mtd fields.
- Define Zimperium vendor and zimperium-mtd format manifests with descriptions and data source metadata.
- Add custom zimperium.mtd field definitions for threat classification, categories, vectors, groups, events, and processes.

Documentation:
- Add Zimperium vendor README describing the platform and its intake formats.
- Add a changelog stub for the zimperium-mtd intake format.

Tests:
- Set up initial test and smart-description artifacts for the zimperium-mtd intake format.

### CODE REVIEW

File: Zimperium/zimperium-mtd/ingest/parser.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        filter: "{{parsed_event.message.generalInfo.externalIp | is_ipaddress}}"
+
+      - set:
+          process.pid: "{{parsed_event.message.processList[0].processId}}"
+          process.parent.pid: "{{parsed_event.message.processList[0].parentProcessId}}"
+          process.name: "{{parsed_event.message.processList[0].processName}}"
+        filter: "{{parsed_event.message.processList | length == 1}}"
+
+      - set:
+          zimperium.mtd.processes: >
```
Comment (LenaigKaliou) [inline Zimperium/zimperium-mtd/ingest/parser.yml:81]: This field does not appears to be working, could you please rework the Jinja ?

**Review by LenaigKaliou — CHANGES_REQUESTED** [review]
Thanks for the work ! Just a small comment

**Review by LenaigKaliou — APPROVED** [review]
LGTM

**Review by squioc — APPROVED** [review]
LGTM
